__tests__/.env.test

```diff
@@ -0,0 +1,5 @@
+GITLAB_URL=https://gitlab.com
```

Comment:
Where is the dotenv being loaded?

Reply:
Good catch. Now being loaded in index.js

Reply:
Why is this needed? It isn't in the existing auth module. I think we can assume that the envvars already being used are being loaded by npme. We shouldn't need the admin API key.